exporter/metric/metric.go

```diff
@@ -21,8 +21,10 @@ import (
 	"strings"
 	"time"
 
-	global "go.opentelemetry.io/otel"
-	apimetric "go.opentelemetry.io/otel/metric"
+	monitoring "cloud.google.com/go/monitoring/apiv3/v2"
```

Comment:
please keep non-opentelemetry imports grouped at the bottom.  This also includes the protobuf import.

Reply:
changed it in all files